src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/clients/RestApiClientV2WithIdleHandle.groovy

```diff
@@ -38,7 +39,7 @@ class RestApiClientV2WithIdleHandle {
                         sleep(1000)
                     }
                 }
-                throw new TimeoutException("Timeout")
+                throw new TimeoutException("Timeout exceeded during request: ${call.request().method()}: ${call.request().url()}")
```

Comment:
For example a package is already running and the user try to execute another package, he gets the information
```
ERROR: Execution has exceeded the configured timeout of 10 seconds
Executing project 'example.pkg' failed!
ERROR: Timeout exceeded during request: PUT: http://localhost:5050/api/v2/configuration
```
I guess this is not a helpful information for the user 🙈

Reply:
Adjusted to ERROR: Timeout exceeded -> no real information gained, since "ERROR: Execution has exceeded the configured timeout of x seconds" already gets logged
alternatively: 
Could add more information in V2 by adding try/catch (timeoutexception) blocks around api calls in RestApiClientV2 -> would add a lot of code for relatively little information
or
Evaluate current call in RestApiClientV2WithIdleHandle.groovy -> also would add quite some lines for relatively little information